src/main/java/seedu/address/model/appointment/Mark.java

```diff
@@ -0,0 +1,70 @@
+package seedu.address.model.appointment;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Note in the appointment
+ * Guarantees: immutable; note is valid as declared in {@link #isValidNote(String)}
+ */
+public class Mark {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Mark should only be true or false, and it should not be blank";
+
+    public final boolean mark;
```

Comment:
should we call this isMarked instead? (or something else since we are using isMarked later)